src/components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -85,6 +94,26 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
 
   const [state, dispatch] = useReducer(resourceFormReducer, initialState);
 
+  const { data: facilitiesResponse } = useQuery<
+    PaginatedResponse<FacilityData>
+  >({
+    queryKey: ["facilities", qParams],
+    queryFn: query.debounced(facilityApi.getAllFacilities, {
+      queryParams: {
+        name: qParams.name,
+        ...(qParams.facility_type && { facility_type: qParams.facility_type }),
+        ...(qParams.organization && {
+          organization: qParams.organization,
+        }),
```

Comment:
why over-complicate things? 🤔 our `makeQuery` will automatically exclude out values that are `undefined`.

<img width="722" alt="image" src="https://github.com/user-attachments/assets/dda63ca3-3e9c-4ca4-85c0-270272c8948c" />


```suggestion
        facility_type: qParams.facility_type,
        organization: qParams.organization,
```